packages/graphql-serve/src/runtime.ts

```diff
@@ -25,19 +33,50 @@ export const createMongoDBClient = async (): Promise<MongoClient> => {
   return client
 }
 
+export function getConflictMapForModels(model: GraphQLSchema, conflictResolutionStrategy: ConflictResolutionStrategy, deltaTTL: number): DataSyncModelConfigMap {
+  const map: DataSyncModelConfigMap = {};
+
+  const dataSyncModelTypes = filterModelTypes(model).filter((type: GraphQLObjectType) => {
+    return !!isDataSyncType(type);
+  })
```

Comment:
Much better. 

I think it could be also written as a oneliner.

```ts
 const dataSyncModelTypes = filterModelTypes(model).filter(isDataSyncType)
```

Bu most important we do not need the `!!` since the function returns a boolean already.

Reply:
Ahh, thanks! :D

Reply:
I think that method should not exist. We need global config in the plugin itself.

Reply:
:-) Yeap. I did not want to make call for having a global default config.

That should solve it. I am sorry @ssd71 for making you do rounds on this. :-(